games-strategy/freeciv/freeciv-3.0.9.ebuild

```diff
@@ -0,0 +1,185 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-3 )
+
+inherit desktop lua-single qmake-utils xdg
+
+MY_PV="${PV/_beta/-beta}"
+MY_PV="${MY_PV/_rc/-RC}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="Multiplayer strategy game (Civilization Clone)"
+HOMEPAGE="https://www.freeciv.org/"
+
+if [[ ${PV} != *_beta* ]] && [[ ${PV} != *_rc* ]] ; then
+	SRC_URI="mirror://sourceforge/freeciv/${MY_P}.tar.xz"
+	KEYWORDS="~amd64 ~ppc64 ~x86"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="aimodules auth dedicated +gtk mapimg modpack mysql nls qt5 readline sdl +server +sound sqlite +system-lua"
+
+REQUIRED_USE="
+	system-lua? ( ${LUA_REQUIRED_USE} )
+	dedicated? ( !gtk !mapimg !modpack !nls !qt5 !sdl !sound )
+	!dedicated? ( || ( gtk qt5 sdl ) )
+"
+
+# postgres isn't yet really supported by upstream
```

Comment:
Is there an issue or a milestone that could be tracked, and linked here?

Reply:
not really - it bugged me when doing the last update, but upstream have gone and made life so much easier for us in the meson-build path by not requiring this info - Given that it'll be released in a month with 3.1.0 I'm going to avoid digging further into it.